README.md

```diff
@@ -1,9 +1,15 @@
-# RSpec tests for your Puppet manifests & modules [![Build Status](https://travis-ci.org/rodjek/rspec-puppet.png)](https://travis-ci.org/rodjek/rspec-puppet)
+# RSpec tests for your Puppet manifests & modules
+[![Build Status](https://travis-ci.org/rodjek/rspec-puppet.png)](https://travis-ci.org/rodjek/rspec-puppet)
+[![Coverage Status](https://coveralls.io/repos/rodjek/rspec-puppet/badge.svg?branch=master)](https://coveralls.io/r/rodjek/rspec-puppet?branch=master)
 
 ## Installation
 
     gem install rspec-puppet
 
+> Note for ruby 1.8 users:  while rspec-puppet itself supports ruby 1.8, you'll
+> need to pin rspec itself to `~> 3.1.0`, as later rspec versions do not work
+> on old rubies anymore.
```

Comment:
Just out of curiosity, what breaks on 1.8 with RSpec 3.1.0+? As to my limited knowledge, RSpec makes its best effort to continue support ruby 1.8.7, sometimes sacrificing conciseness.

Reply:
No idea what broke 4 years ago. Apologies :-|

To my relief we're off 1.8 now for a long time.